ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/txselection/BlockTransactionSelector.java

```diff
@@ -301,17 +345,47 @@ private TransactionSelectionResult handleTransactionSelected(
     final long blobGasUsed =
         blockSelectionContext.gasCalculator().blobGasCost(transaction.getBlobCount());
 
-    final TransactionReceipt receipt =
-        transactionReceiptFactory.create(
-            transaction.getType(), processingResult, worldState, cumulativeGasUsed);
+    final boolean tooLate;
 
-    logTransactionSelection(pendingTransaction.getTransaction());
```

Comment:
does this still get logged if the tx is selected?

Reply:
yes, since it was only use in one place, I just moved it inline